u_tls_extensions.go

```diff
@@ -727,8 +727,11 @@ func (e *FakeCertCompressionAlgsExtension) Read(b []byte) (int, error) {
 		return 0, errors.New("too many certificate compression methods")
 	}
 
+	// Extension data length.
 	b[2] = byte((extLen + 1) >> 8)
 	b[3] = byte((extLen + 1) & 0xff)
+
+	// Methods length.
```

Comment:
The structure of the code here confused me, so I added a couple comments and a newline